invisible_cities/calib/calib_functions.py

```diff
@@ -136,9 +141,78 @@ def scaled_spectrum(x, mu):
     return scaled_spectrum
 
 
+def seeds_db(sensor_type, run_no, n_chann):
+    """
+    Take gain and sigma values of previous runs in the database
+    to use them as seeds.
+    """
+    if sensor_type == 'sipm':
```

Comment:
You should relally use some kind of symbol, rather than a string, to denote values belonging to a limited set of possibilitiies. Perhaps an `Enum`.

Reply:
What do you mean by a kind of symbol?

Reply:
Strings are types with an almost infinite number of values. By accepting strings you are implying that any of these might be OK. In this context, only a very limited set of values is acceptable, so accept a type that has only that limited set.

That way, there is automatic documentation of what the allowed values are, inadvertent misspelling is more difficult, static tools can more easily help you out, etc., etc..

Furthermore, strings usually need to be compared by looking through all their elements, one by one, until a mismatch is found, or the end is reached. Symbols, can be compared by identity, so comparison is faster.

I know that it is widespread practice in Python to accept strings where symbols would be *The Right Thing*, but that doesn't mean we should perpetuate bad practices. This bad practice was understandable (and maybe even excusable) when Python didn't have anything resembling a symbol type that was easy to use, but now that we have `enum` there really is no excuse.